mmdet/apis/ote/apis/detection/config_utils.py

```diff
@@ -78,8 +79,8 @@ def set_hyperparams(config: Config, hyperparams: OTEDetectionConfig):
     else:  # Epoch based runner
         config.runner.max_epochs = total_iterations
     num_checkpoints = int(hyperparams.learning_parameters.num_checkpoints)
-    config.evaluation.interval = total_iterations // num_checkpoints
-    config.checkpoint_config.interval = total_iterations // num_checkpoints
+    config.evaluation.interval = math.ceil(total_iterations / num_checkpoints)
+    config.checkpoint_config.interval = math.ceil(total_iterations / num_checkpoints)
```

Comment:
What are the pros of it?

Reply:
If we use in some micro-training `total_iterations=3`, we could have `total_iterations < num_checkpoints`. In this case `config.evaluation.interval` was `0` and error happened.

Reply:
I see. Thanks.